### PR TITLE
Fix section structure error (6. Cross-domain Collaboration, 7. System Integration)

### PR DESCRIPTION
**Pull request by k-toumura**
(related issue/PR: #628, #771)
I didn't notice in on the Arch call in May 24th, some structure of section seems broken.
There is extra `</section>` tag on after "5.9 Virtual Things": https://github.com/w3c/wot-architecture/blob/02b9aeaf3cffe29b72c4522362c9380aca5c2eb3/index.html#L1568-L1571

This extra tag accidentally makes the subsequent "Cross-domain Collaboration" and "System Integration" subsection independent from the Section 5 "Common Deployment Patterns".

This PR remove the extra section tag, and also remove introduction text for "Cross-domain Collaboration" and "System Integration" (and also fix section ID).   The resulting section structure is as follows:
```
  ...
5. Common Deployment Patterns
5.1 Telemetry
  ...
5.9 Virtual Things
5.10 Cross-domain Collaboration
5.11 System Integration
6. Abstract WoT System Architecture
  ...
```

(if the original section structure is intentional, please close this PR)

***
<a href="https://pr-preview.s3.amazonaws.com/k-toumura/wot-architecture/pull/774.html" title="Last updated on May 25, 2022, 12:17 AM UTC (d9b98ea)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-architecture/774/02b9aea...k-toumura:d9b98ea.html" title="Last updated on May 25, 2022, 12:17 AM UTC (d9b98ea)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -380,13 +380,6 @@ <h1>Introduction</h1>
       <li>a set of common deployment patterns in
         <a href="#sec-common-deployment-patterns"></a>,
       </li>
-      <li>
```
Comment (mlagally) [inline index.html:383]: Why did you remove that paragraph?

**Review by mlagally — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (k-toumura) [reply index.html:383]: Because "Introduction" contains a description of sections, not subsections.   
"Cross domain collaboration" and "System Integration" section are moved to subsections in Section 5
(5.10 and 5.11, respectively).
